app.R

```diff
@@ -655,7 +654,8 @@ server <- function(input, output) {
       }
     }
     else
-      a <- read.table(file = file1$datapath, sep = input$sep, header = input$header, stringsAsFactors = FALSE)
+      a <- rio::import(file1$datapath, header = input$header, stringsAsFactors = FALSE)
```

Comment:
Does this need any tests (or are there some example files I can use to test manually?)

Reply:
I believe you have already tested this manually, but the default files can be opened in excel and saved as .xlsx